### PR TITLE
Avoid including `group` in PR titles twice

### PR DESCRIPTION
**Pull request by jurre**
image

### CODE REVIEW

**Review by honeyankit — APPROVED** [review]

**Comment by abdulapopoola** [comment]
Also, do we need tests for this?

File: updater/lib/dependabot/dependency_group_engine.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@ def self.from_job_config(job:) # rubocop:disable Metrics/PerceivedComplexity
         # - This is a security update and there are multiple dependencies passed in
         # Since there are no groups, the default behavior is to group all dependencies, so create a fake group.
         job.dependency_groups << {
-          "name" => "#{job.package_manager} group",
```
Comment (jurre) [inline updater/lib/dependabot/dependency_group_engine.rb:35]: @abdulapopoola this is where the second group was coming from

File: common/lib/dependabot/pull_request_creator/message_builder.rb [hunk 2]
```diff
@@ -242,7 +242,7 @@ def group_pr_name
         updates = dependencies.map(&:name).uniq.count
 
         if source.directories
-          "bump the #{T.must(dependency_group).name} across #{T.must(directories_with_updates).count} " \
+          "bump the #{T.must(dependency_group).name} group across #{T.must(directories_with_updates).count} " \
```
[the remark this answers is not in the document]
Comment (jurre) [reply common/lib/dependabot/pull_request_creator/message_builder.rb:245]: See https://github.com/dependabot/dependabot-core/pull/9384#discussion_r1543307831

File: updater/lib/dependabot/dependency_group_engine.rb (hunk 1 above)
Comment (abdulapopoola) [reply updater/lib/dependabot/dependency_group_engine.rb:35]: the line above does add a new "group" entry though;

**Review by abdulapopoola — APPROVED** [review]

**Comment by jurre** [comment]
I think I'll need to regenerate all the smoke tests after this 😬

**Comment by jurre** [comment]
> Also, do we need tests for this?

I _think_ all the existing tests should now cover this, especially after the smoke tests are updated

**Pull request merged**